package.json

```diff
@@ -21,6 +21,9 @@
   },
   "homepage": "https://github.com/Zhibul-Alexander/scheduler-be#readme",
   "dependencies": {
+    "@types/bcryptjs": "^2.4.2",
+    "@types/cors": "^2.8.13",
+    "@types/jsonwebtoken": "^9.0.1",
```

Comment:
What do you think to move it to devDependencies?

Reply:
yes, you need to migrate those dependencies